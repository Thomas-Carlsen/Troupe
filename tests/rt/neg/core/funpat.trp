let fun isOne 1 = true
    and isTwo 2 = true 
in isTwo () 
end
