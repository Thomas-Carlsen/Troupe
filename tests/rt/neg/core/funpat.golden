2019-03-27T20:06:28.508Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mRuntime error in thread a92ffcf3-8a74-4f42-b8ae-114a40873937@{}%{}[39m
[31m>> pattern match failure in function isTwo at :2:9[39m
